src/content/docs/concepts/security/index.mdx

```diff
@@ -0,0 +1,122 @@
+---
+title: Security
+---
+
+import { CardGrid, LinkCard } from '@astrojs/starlight/components';
+
+This page seeks to explain the high-level concepts and security features
+at the core of Tauri's design and ecosystem that make you, your apps, and your users safer by default.
+
+It also contains advice for best practices, how to disclose vulnerabilities to us
+and references to in-detail concept explanations.
+
+:::note
+
+It is important to remember that the security of your Tauri application is the sum of the overall security
+of Tauri itself, all Rust and npm dependencies, your code, and the devices that run the final application.
+The Tauri Team does their best to do their part,
+the security community does its part, and you too should follow a few important best practices.
+
+:::
+
+## Security Boundaries
+
+Tauri's security model differentiates between rust code written for the application's
+core and frontend code written in any framework or language understood by the system
+WebView.
+
+The IPC layer is the bridge for communication between these two trust groups and ensures
+that boundaries are not broken.
+
+
+![IPC Diagram](@assets/concepts/security-boundaries.svg)
+
+Any code executed by the plugins or the application core has full
+access to all available system resources and is not constrained.
+
+Any code executed in the WebView has only access to system resources
+via the well defined [IPC](/concepts/inter-process-communication) layer. The access of application core commands is configured
+and constrained via capabilities defined in the application configuration.
+The individual command implementations enforce the optional fine grained access level,
+which are also defined in the capabilities configuration.
+
+Learn more about the individual components and boundary
+enforcement:
+
+<CardGrid>
+	<LinkCard
```

Comment:
After reading it I find the natural order of the subsections should be:
1. Permissions
2. Scopes
3. Capabilities
4. Runtime Authority